index.html

```diff
@@ -224,9 +224,9 @@ <h2 data-i18n="dgc-getting-started" class="page-section-heading text-center text
                         <br><br>
                         <span data-i18n="dgc-exchanges-modal-tipped">You can get tipped Dogecoin by participating in <a class="btn btn-sm btn-outline-secondary js-scroll-trigger" href="#about">our community</a>
                             <br>
-                            If that's not your style, another way to have your first Dogecoin is to participate at Folding@Home.</span>
+                            If that's not your style, another way to have your first Dogecoin is to participate at <a href="https://www.dogecoinfah.com/">Dogecoin Folding@Home</a>.</span>
                         <br><br>
-                        <span data-i18n="dgc-exchanges-modal-fah-faq">Folding@Home (FaH) is a distributed computing project. You can join team dogefolders, bring in your computing power and receive some Doges for your efforts.</span>
+                        <span data-i18n="dgc-exchanges-modal-fah-faq">Folding@Home (FaH) is a distributed computing project. You can join team dogefolders, bring in your computing power and receive some Dogecoins for your efforts. Please note the team rewards you with Dogecoins, Folding@Home is not affiliated with Dogecoin.</span>
```

Comment:
Suggesting: 
Please note any reward is paid by the Dogecoin Folding@Home team. The Folding@Home project is not affiliated with Dogecoin.

Reply:
For German:
Bitte beachte, dass eventuelle Belohnungen vom Dogecoin Folding@Home Team gezahlt werden. Das Folding@Home Projekt steht in keiner Beziehung zu Dogecoin.